index.js

```diff
@@ -0,0 +1,164 @@
+let taskArr = [];
+let start = 0;
+let current = 0;
```

Comment:
Please move these variables inside `drag` function, because these don't need to have global scope, following the JavaScript best practices.